torch/csrc/jit/mobile/flatbuffer_loader.h

```diff
@@ -50,5 +50,64 @@ TORCH_API mobile::Module load_mobile_module_from_file(
     const std::string& filename,
     c10::optional<at::Device> device = c10::nullopt);
 
+class FlatbufferLoader {
+ public:
+  FlatbufferLoader();
+
+  typedef IValue (
+      *IValueParser)(FlatbufferLoader*, const mobile::serialization::IValue*);
```

Comment:
consider `typedef IValue (*IValueParser)(FlatbufferLoader&, const mobile::serialization::IValue&)`. Previously pointer passing was just an implementation detail in loader, but now since we allow overriders here, it's better to make the contract clear that both arguments are non-null values.

Reply:
done.